pandas/_testing/asserters.py

```diff
@@ -609,6 +609,43 @@ def assert_interval_array_equal(
     assert_attr_equal("inclusive", left, right, obj=obj)
 
 
+def assert_interval_array_almost_equal(
```

Comment:
If you want to make this public, we should discuss first. If not, you don’t have to use versionaddeds and stuff like that in the docstring

Reply:
Thanks for the comment @phofl. What do you mean by make public?

Reply:
Should this be accessible by our users?

Reply:
Oh, no. I'll take out the wersionaddeds from the docstring. Is there anything else I'd need to remove?

Reply:
No, should be fine then